packages/vscode-ui5-language-assistant/test/suite/validations/flex-disabled/flex-disabled-spec.ts

```diff
@@ -0,0 +1,54 @@
+import * as vscode from "vscode";
+import { resolve, dirname } from "path";
+import { deactivate } from "../../../../src/extension";
+import { setXMLContent, testDiagnostics, sleep } from "../../test-utils";
+
+const pkgJsonPath = require.resolve(
+  "vscode-ui5-language-assistant/package.json"
+);
+
+const rootPkgFolder = dirname(pkgJsonPath);
+const testFixtureValidationsFolder = resolve(
+  rootPkgFolder,
+  "test",
+  "test-fixtures",
+  "validations"
+);
+
+describe("the Language Server Client Validations Integration Tests - Flex Disabled", () => {
+  const testFolderPath = resolve(
+    testFixtureValidationsFolder,
+    "flex-disabled-spec"
+  );
+
+  const testFolderUri = vscode.Uri.file(testFolderPath);
+  const xmlPath = resolve(testFolderPath, "test.view.xml");
+  const xmlUri = vscode.Uri.file(xmlPath);
+  const manifestPath = resolve(testFolderPath, "manifest.json");
+  const manifestUri = vscode.Uri.file(manifestPath);
+
+  before(async () => {
+    await vscode.commands.executeCommand("vscode.openFolder", testFolderUri);
+    await vscode.window.showTextDocument(xmlUri);
+    await vscode.workspace.openTextDocument(manifestUri);
+    // Explicitly wait for extension to load
+    await sleep(1000);
```

Comment:
this number (1000) should be a const, e.g: `EXTENSION_START_TIMEOUT = 5000` in the test utils